src/tracks/InstrumentTrack.cpp

```diff
@@ -35,6 +35,7 @@
 #include <QMessageBox>
 #include <QMdiSubWindow>
 #include <QPainter>
+#include <QSignalMapper>
```

Comment:
You can remove this as well.

Reply:
Thanks! Forgot to remove it too :facepalm: